lib/l10n/app_en.arb

```diff
@@ -17,5 +17,369 @@
+    "mother": "Mother",
+    "@mother": {
+        "Mother": "Mother"
+    },
+
+    "brother": "Brother",
+    "@brother": {
+        "description": "Brother"
+    },
+
+    "sister": "Sister",
+    "@sister": {
+        "description": "Sister"
+    },
+
+    "edit": "Edit",
+    "@edit": {
+        "description": "Edit"
+    },
+
+    "delete": "Delete",
+    "@delete": {
+        "description": "Delete"
+    },
+
+    "mister": "Mr.",
+    "@mister": {
+        "description": "Mr."
+    },
+
+    "mistress": "Mrs.",
+    "@mistress": {
+        "description": "Mrs."
+    },
+
+    "choosefamilymember": "Choose your family member",
+    "@choosefamilymember": {
+        "description": "Choose your family member"
+    },
+
+    "title": "Title",
+    "@title": {
+        "description": "Title"
+    },
+
+    "firstname": "First Name",
+    "@firstname": {
+        "description": "First Name"
+    },
+
+    "familyname": "Family Name",
+    "@familyname": {
+        "description": "Family Name"
+    },
+
+    "email": "E-Mail",
+    "@email": {
+        "description": "E-Mail"
+    },
+
+    "phonenumber": "Telephone Number",
+    "@phonenumber": {
+        "description": "Telephone Number"
+    },
+
+    "address": "Address",
+    "@address": {
+        "description": "Address"
+    },
+
+    "abort": "Abort",
+    "@abort": {
+        "description": "Abort"
+    },
+
+    "proceed": "Continue",
+    "@proceed": {
+        "description": "Continue"
+    },
+
+    "myentries": "My Entries",
+    "@myentries": {
+        "description": "My Entries"
+    },
+
+    "startentry": "Start Entry",
+    "@startentry": {
+        "description": "Start Entry"
+    },
+
+    "submitdata": "The entered will be submitted.",
+    "@submitdata": {
+        "description": "The entered will be submitted."
+    },
+
+    "husband": "Husband",
+    "@husband": {
+        "description": "Husband"
+    },
+
+    "wife": "Wife",
+    "@wife": {
+        "description": "Wife"
+    },
+
+    "entryemail": "Please enter the e-mail-address!",
+    "@entryemail": {
+        "description": "Please enter the e-mail-address!"
+    },
+
+    "entryvalidemail": "Please enter a valid e-mail-address!",
+    "@entryvalidemail": {
+        "description": ""
+    },
+
+    "entryfirstname": "Please enter the first name!",
+    "@entryfirstname": {
+        "description": "Please enter the first name"
+    },
+
+    "entryfamilyname": "Please enter the family name!",
+    "@entryfamilyname": {
+        "description": "Please enter the family name"
+    },
+
+    "entryaddress": "Please enter the address!",
+    "@entryaddress": {
+        "description": "Please enter the family name"
+    },
+
+    "entryphonenumber": "Please enter the phone number!",
+    "@entryphonenumber": {
+        "description": "Please enter the phone number!"
+    },
+
+    "myphysicians": "My Physicians",
+    "@myphysicians": {
+        "description": "My Physicians"
+    },
+
+    "practice": "Practice",
```

Comment:
Is this the correct term in the context? The german translations "Praxis" could mean multiple things.
Maybe "surgery" would fit better.

Reply:
Well, as far as I know "surgery" is just a room where a physician does his job. In my opinion, "practice" would be more appropriate. I let it open for discussion :)

Reply:
Afaik "practice" can be used, but it is ambiguous.
But we could leave it as it is and change it later if someone complains about it.